prime-dotnet-webapi/Controllers/EnrolmentsController.cs

```diff
@@ -176,13 +178,13 @@ private bool BelongsToEnrollee(Enrolment enrolment)
             //if user is not ADMIN, check that user belongs to the enrolment
             if (!belongsToEnrollee)
             {
-                var PrimeUserId = PrimeUtils.PrimeUserId(User);
-                belongsToEnrollee = PrimeUserId != null
+                Guid PrimeUserId = PrimeUtils.PrimeUserId(User);
+                belongsToEnrollee = !Guid.Empty.Equals(PrimeUserId)
```

Comment:
I dislike this pattern - Is this there a better way?

Reply:
I changed the order of the check to make this a little easier to read.  The issue is that Guid cannot be null, so they use a special Guid.Empty to represent that.  I also added some comments to the code to help explain it better.